seong-geun/db/migrations/20230324021716_create_users_table.sql

```diff
@@ -0,0 +1,13 @@
+-- migrate:up
+CREATE TABLE users (
+    userId VARCHAR(100) NOT NULL PRIMARY KEY,
+    userProfileImage varchar(1000) NULL,
+    userName VARCHAR(100),
+    age INT NULL,
+    email VARCHAR(1000) NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
```

Comment:
- PK를 의미하는 id는 일반적으로 INT를 사용하고, `id`라는 컬럼명을 사용합니다.
- mysql의 컬럼명은 주로 스네이크 케이스로 컬럼명을 만듭니다.
- profile_image의 경우 이미 user table에 있는 컬럼명이므로 user의 profile_image라는 의미가 충분히 있습니다.
- username같은 경우 우리가 흔히 말하는 계정의 ID를 의미하는 컬럼으로 쓰입니다. 혹시나 사람의 이름을 나타내고 싶었던 거였으면 `name`으로 컬럼명을 바꿔주시면 좋을 것 같습니다.
```suggestion
CREATE TABLE users (
    id INT NOT NULL PRIMARY KEY,
    profile_image varchar(1000) NULL,
    username VARCHAR(100),
    age INT NULL,
    email VARCHAR(1000) NOT NULL,
    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
```